src/dialogs/transcodedialog.ui

```diff
@@ -140,6 +137,61 @@
      </property>
     </widget>
    </item>
+   <item>
+    <widget class="QWidget" name="advancedWidget" native="true">
+     <layout class="QVBoxLayout" name="verticalLayout_2">
+      <item>
+       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
+        <item row="1" column="1">
+         <widget class="QCheckBox" name="fpsCheckBox">
+          <property name="toolTip">
+           <string>Enable frame rate override.</string>
+          </property>
+          <property name="text">
+           <string>Override Frame Rate</string>

```

Comment:
Please change the casing on this label to uppercase first word only. See https://shotcut.org/notes/ui-conventions/

Reply:
fixed